### DISCUSSION TITLE
SVTR trained model

### DISCUSSION THREAD

**Discussion by senstar-hsoleimani** [PaddlePaddle/PaddleOCR#8364] [Q&A]
Is SVRT trained model available?  In my Python code, I like to use the .pt or .pb   format of the trained version of SVRT.

**Comment by andyjiang1116** [comment]
model link is https://paddleocr.bj.bcebos.com/PP-OCRv3/chinese/rec_svtr_tiny_none_ctc_en_train.tar
refer doc https://github.com/PaddlePaddle/PaddleOCR/blob/release/2.6/doc/doc_en/algorithm_rec_svtr_en.md
